File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    string even = "", odd = "";
    for(int i=0; i< num_list.size();i++){
        if(num_list[i]%2 == 0){
            even += ('0' + num_list[i]);
        }
        else{
            odd += ('0' + num_list[i]);
        }
        
    }
    
    return stoi(even) + stoi(odd);
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        String even = "", odd = "";
        for(int num: num_list){
            if(num%2 == 0)
                even += (char)('0' + num);
            else
                odd += (char)('0' + num);
        }
        
        return Integer.parseInt(even) +  Integer.parseInt(odd);
    }
}